sycl/doc/extensions/PlatformContext/PlatformContext.adoc

```diff
@@ -0,0 +1,29 @@
+= SYCL(TM) Proposals: Platform Default Contexts
+James Brodman <james.brodman@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+:y: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Platform Default Contexts
+
+This extension adds the notion of a default SYCL context per SYCL platform. The default context for each platform contains all devices in the platform.
+
+The platform class gains one new method:
+
+[cols="^60a,40"]
```

Comment:
It seems now to be complaint with SYCL2020 the extension should define a feature macro and prefix public methods with `ext_`.

Reply:
ok.